src/main/com/yubico/yubioath/MainActivity.java

```diff
@@ -125,17 +120,10 @@ public void onResume() {
             }
             DialogFragment dialog = new EnableNfcDialog();
             dialog.show(ft, "dialog");
-        } else if (totpListener != null) {
-            Intent intent = getIntent();
-            if (readOnResume && intent.getAction().equals(NfcAdapter.ACTION_NDEF_DISCOVERED)) {
```

Comment:
@iDJH @Franzaine I think the removal of this is what discards the tag when the app is invoked via intent as described in #27 . My bad, will provide a fix.